src/coordination/coordinator_instance.cpp

```diff
@@ -95,7 +95,8 @@ CoordinatorInstance::CoordinatorInstance(CoordinatorInstanceInitConfig const &co
               repl_instances_.clear();
               spdlog::info("Stopped all replication instance frequent checks.");
             });
-          })) {
+          })),
+      config_{config} {
```

Comment:
In general this constructor is very hard to read. Could we break it down somehow?

Reply:
Agree, will think about it